setup.py

```diff
@@ -183,13 +180,13 @@
       version=__version__,
       description="NumPy-like API accelerated with SYCL",
       long_description=__readme_file__,
+      long_description_content_type="text/markdown",
       author="Intel Corporation",
-      author_email="Intel Corporation",
       maintainer="Intel Corp.",
       maintainer_email="scripting@intel.com",
       url="https://intelpython.github.io/dpnp/",
       download_url="https://github.com/IntelPython/dpnp",
-      license=__license_file__,
+      license='BSD',
```

Comment:
Is this change related to "wheel testing"?

Reply:
Yes, changing setup.py format is required to build a good wheel which can be uploaded to Pypi